projects/media-viewer/src/lib/toolbar/media-viewer-toolbar/media-viewer-toolbar.component.html

```diff
@@ -0,0 +1,193 @@
+<div class="mv-toolbar__container">
```

Comment:
it seems like you could split this into 3 components to simplify the html: mainToolbar, moreOptions, menuItems

Reply:
Related to code complexity. Potential for additional technical debt if a different approach is adopted.